### PR TITLE
Work 25.02

### PR DESCRIPTION
**Pull request by sotickmars**
На проверку Курсовая

### CODE REVIEW

File: project/assets/less/common/mixins.less [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  font-size: @fs;
+}
+
+.width-height(@w:100%, @h:100%) {
+  width: @w;
+  height: @h;
+}
+
+.transf-dur(@td:0.4s) {
+  transition-duration: @td;
```
Comment (tpotapchik) [inline project/assets/less/common/mixins.less]: This mixin has no sense

File: project/assets/less/layout/styles.less [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+  color: #fff;
+  line-height: 1.092;
+  .font-main(@ff: @font-additional, @fw:700, @fs:65px);
+  .margin-top-bot();
+}
+
+.content {
+  font-family: @font-default;
+  line-height: 1.5;
+  .font-main(@fs:22px);
```
Comment (tpotapchik) [inline project/assets/less/layout/styles.less]: Better to place mixins at the end

File: project/assets/less/layout/styles.less [hunk 3]
```diff
@@ -0,0 +1,472 @@
...
+  padding-bottom: 80px;
+  .margin-top-bot();
+}
+
+.button-find {
+  height: 45px;
+  width: 200px;
+  display: block;
+  border: none;
+  color: inherit;
```
Comment (tpotapchik) [inline project/assets/less/layout/styles.less:118]: Not a good way if we need to add another wrapper

File: project/assets/less/layout/styles.less [hunk 4]
```diff
@@ -0,0 +1,472 @@
...
+  padding-bottom: 120px;
+}
+
+.services-items {
+  width: 33%;
+  text-align: center;
+  padding-left: 45px;
+  padding-right: 45px;
+  box-sizing: border-box;
+  margin-top: 50px;
```
Comment (tpotapchik) [inline project/assets/less/layout/styles.less:148]: Better to add to the whole wrapp - not a one of the 3 items

File: project/assets/less/layout/styles.less [hunk 5]
```diff
@@ -0,0 +1,472 @@
...
+  color: #6c7279;
+  .font-main();
+  .margin-top-bot();
+}
+
+/***********IMG-CONTAINER******************/
+
+.wrapper-images {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (tpotapchik) [inline project/assets/less/layout/styles.less]: You can add it to mixin for example

File: project/assets/less/layout/styles.less [hunk 6]
```diff
@@ -0,0 +1,472 @@
...
+.inner-command {
+  width: 23%;
+}
+
+.img-command {
+  width: 100%;
+  height: 270px;
+  img {
+    object-fit: cover;
+    width: 100%;
```
Comment (tpotapchik) [inline project/assets/less/layout/styles.less]: Forgot mixin

**Review by tpotapchik — APPROVED** [review]

File: project/assets/less/layout/media.less [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+    }
+  }
+
+  .inner-wrap-command {
+    flex-wrap: wrap;
+  }
+
+  .inner-command {
+    width: 47%;
+    &:nth-of-type(-n + 2) {
```
Comment (tpotapchik) [inline project/assets/less/layout/media.less]: First-child?

File: project/assets/less/layout/media.less [hunk 8]
```diff
@@ -0,0 +1,230 @@
...
+
+  .inner-icons {
+    margin-top: 10px;
+  }
+
+  .wrap-quote {
+    display: block;
+    img {
+      object-position: right;
+    }
```
Comment (tpotapchik) [inline project/assets/less/layout/media.less:45]: Better to define x and y also

File: project/assets/less/layout/styles.less (hunk 4 above)
Comment (tpotapchik) [reply project/assets/less/layout/styles.less:148]: ?

**Review by tpotapchik — APPROVED** [review]

**Comment by sotickmars** [comment]
Курсовой проект HTML CSS

**Pull request merged**